src/sass/base/_images.scss

```diff
@@ -0,0 +1,11 @@
+.image-person-placeholder {
```

Comment:
I would combine these together so you only have to use the property once:

```
.image {
    &-person, &-organization {
        &-placeholder {
            vertical-align: middle;
        }
    }
}
```
I think that is how you would do the Sass. Have not worked with Sass in awhile but the point would be to have both classes share the property and value in one rule